### ISSUE TITLE
Facebook is on something

### ISSUE THREAD

**Issue by cspang1** [cspang1/sm-parser#9]
```
{
  "group_posts": {
    "activity_log_data": [
      {
        "timestamp": 1550977333,
        "data": [
          {
            "comment": {
              "timestamp": 1550977333,
              "comment": "redacted shenanigans",
              "author": "Facebook",
              "group": "Why Facebook Why"
            }
          }
        ],
        "title": "Facebook hates us"
      },
    ]
  }
}
```

**Comment by cspang1** [comment]
`your_posts_and_comments_in_groups.json`

**Comment by cspang1** [comment]
Fixed w/ commit caff520b8ee33716b1a7853ce0ad27c475e5e632
